### ISSUE TITLE
Feature request: Max video bitrate setting

### ISSUE THREAD

**Issue by someone30** [fgl27/smarttv-twitch#39]
Please implement an  "Max video bitrate" setting, which presets the video quality to e.g. 720p60 when opening a stream/broadcast. If the chosen bitrate is not available downgrade/upgrade (as a suboption?) to the closest available.

**Comment by fgl27** [comment]
On settings is something that can be added, but will only show in settings.

**Comment by someone30** [comment]
Yes, the settings section would be the perfect place for this. Thank you!
